common/persistence/cassandra/matching_task_store.go

```diff
@@ -481,28 +481,6 @@ func (d *MatchingTaskStore) GetTasks(
 	return response, nil
 }
 
-// CompleteTask delete a task
-func (d *MatchingTaskStore) CompleteTask(
-	ctx context.Context,
-	request *p.CompleteTaskRequest,
-) error {
-	tli := request.TaskQueue
-	query := d.Session.Query(templateCompleteTaskQuery,
```

Comment:
keep going.. you can delete this query

Reply:
Not sure if by this you meant to delete `templateCompleteTaskQuery` or not. Noticed it is also being used inside of `GetTasks`.